### PR TITLE
Add compact 3 node cluster jobs

### PR DESCRIPTION
**Pull request by shaozha**
Add aws-ipi-compact for arm/amd to:

    4.13 nightly

Resolve issue: https://issues.redhat.com/browse/OCPQE-13515

### CODE REVIEW

File: ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/compact/cucushift-installer-rehearse-aws-ipi-compact-workflow.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+workflow:
+  as: cucushift-installer-rehearse-aws-ipi-compact
```
Comment (yunjiang29) [inline ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/compact/cucushift-installer-rehearse-aws-ipi-compact-workflow.yaml:2]: you don't need to define this workflow again, #36095 has been merged, have you rebased code from upstream to your local branch?

**Comment by yunjiang29** [comment]
@shaozha only following files need to be updated:
* `ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13-periodics.yaml`
* `ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml`
* `ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml`

File: ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/compact/cucushift-installer-rehearse-aws-ipi-compact-workflow.yaml (hunk 1 above)
Comment (shaozha) [reply ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/compact/cucushift-installer-rehearse-aws-ipi-compact-workflow.yaml:2]: I haven't rebased code.

**Comment by shaozha** [comment]
@yunjiang29 Thank you, I get it.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml [hunk 2]
```diff
@@ -192,6 +192,37 @@ tests:
...
+  cron: 7 1 23,7 * *
+  steps:
+    cluster_profile: aws-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: m6g.xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
+      E2E_RUN_TAGS: '@arm64 and @aws-ipi and @disconnected and @network-ovnkubernetes
```
Comment (yunjiang29) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml:205]: this is not a disconnected cluster, `@disconnected` should be removed.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 3]
```diff
@@ -217,6 +217,32 @@ tests:
...
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-aws-ipi-compact
+- as: aws-ipi-compact-p3-f28-destructive
+  cron: 16 2 15 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@aws-ipi and @disconnected and @network-ovnkubernetes and not
```
Comment (yunjiang29) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:239]: this is not a disconnected cluster, `@disconnected` should be removed.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 4]
```diff
@@ -217,6 +217,32 @@ tests:
...
+- as: aws-ipi-compact-p2-f14
+  cron: 12 21 1,16 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@aws-ipi and @disconnected and @network-ovnkubernetes and not
+        @fips'
+      TAG_VERSION: '@4.13'
+      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
```
Comment (yunjiang29) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:229]: I don't think `TEST_FILTERS ` is required for this profile.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 5]
```diff
@@ -217,6 +217,32 @@ tests:
...
+- as: aws-ipi-compact-p3-f28-destructive
+  cron: 16 2 15 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@aws-ipi and @disconnected and @network-ovnkubernetes and not
+        @fips and @destructive'
+      TAG_VERSION: '@4.13'
+      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;Serial;Disruptive;~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
```
Comment (yunjiang29) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:242]: same as https://github.com/openshift/release/pull/36163/files#r1101178053

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml [hunk 6]
```diff
@@ -192,6 +192,37 @@ tests:
...
+        and not @fips'
+      OCP_ARCH: arm64
+      TAG_VERSION: '@4.13'
+      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
```
Comment (yunjiang29) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml:209]: same as https://github.com/openshift/release/pull/36163/files#r1101178053

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml [hunk 7]
```diff
@@ -192,6 +192,37 @@ tests:
...
+- as: aws-ipi-compact-p3-f28-destructive
+  cron: 19 14 6 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@arm64 and @aws-ipi and @disconnected and @network-ovnkubernetes
+        and not @fips and @destructive'
+      TAG_VERSION: '@4.13'
+      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;Serial;Disruptive;~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
```
Comment (yunjiang29) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml:222]: same as https://github.com/openshift/release/pull/36163/files#r1101178053

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml (hunk 4 above)
Comment (shaozha) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:229]: I noticed that there have TEST_FILTERS in other profile in this file.So I thought it was necessary.
I can try to remove it.

**Comment by yunjiang29** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-arm64-nightly-aws-ipi-compact-p2-f14

**Comment by yunjiang29** [comment]
@jianlinliu @liangxia PATL

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 8]
```diff
@@ -217,6 +217,28 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-cco-manual-security-token-service-with-ep-sts-ec2-elb
+- as: aws-ipi-compact-p2-f14
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:220]: It's suggested to sort `aws-ipi-compact*` between `aws-ipi-byo*` and `aws-ipi-disconnected*`

Comment (shaozha) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml]: Thank you for your suggestion. Both profiles have been updated.

**Comment by shaozha** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-arm64-nightly-aws-ipi-compact-p2-f14

**Comment by liangxia** [comment]
@shaozha   Good job!

/approve

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml [hunk 9]
```diff
@@ -161,6 +161,34 @@ tests:
...
+      TAG_VERSION: '@4.13'
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-aws-ipi-compact
+- as: aws-ipi-compact-p3-f28-destructive
+  cron: 19 14 6 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml:185]: Missing `COMPUTE_NODE_TYPE`, `CONTROL_PLANE_INSTANCE_TYPE` and `OCP_ARCH`, right?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml (hunk 3 above)
Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml]: @shaozha pls update the PR's initial description accordingly.

**Comment by jianlinliu** [comment]
Pls squash the multiple commits into one.

**Comment by liangxia** [comment]
Need to fetch upstream and rebase.

**Comment by jianlinliu** [comment]
@shaozha from https://github.com/openshift/release/pull/36163/commits, it shows 4 commits, pls squash them into one.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml (hunk 9 above)
Comment (shaozha) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml:185]: Yes, thank you for your suggestion. I have modified it.

**Comment by shaozha** [comment]
@jianlinliu @yunjiang29 After I sync fork from upstream, I can't find this new commit when I use "git log". So I can't squash the two commits into one. Do you have any suggestions？

**Comment by jianlinliu** [comment]
@shaozha pls also check https://github.com/openshift/release/pull/36163#discussion_r1107945989

**Pull request merged**